config/default.json

```diff
@@ -3,8 +3,33 @@
 	"port": 3030,
 	"mongodb": "mongodb://localhost:27017/schulcloud",
 	"public": "../public/",
-	"auth": {
-		"secret": "FQ5lS1j41dcZdaPFPP3VRPH6W2bt8Z19bmvcVz+8SH8PtfVpVTl/p+9D0jIdRhwC1UrWg69lTU9koFgugf9oZg=="
+	"authentication": {
+		"entity": "account",
+		"service": "accounts",
+		"secret": "FQ5lS1j41dcZdaPFPP3VRPH6W2bt8Z19bmvcVz+8SH8PtfVpVTl/p+9D0jIdRhwC1UrWg69lTU9koFgugf9oZg==",
+		"authStrategies": [ "jwt", "local", "ldap", "moodle", "iserv" ],
+		"jwtOptions": {
+			"header": { "typ": "access" },
+			"audience": "https://schul-cloud.org",
+			"issuer": "feathers",
+			"algorithm": "HS256",
+			"expiresIn": "30d"
+		},
+		"local": {
```

Comment:
It is used for what? Local authentication?

Reply:
Yes, I will discuss the configuration with Parnswir and the DevOps